ui/.build/src/main.ts

```diff
@@ -19,8 +19,8 @@ const longArgs = [
   '--prod',
   '--split',
   '--debug',
+  '--rgb',
```

Comment:
building with --rgb will show mixed colors in rgb format in browser dev tools, but it will also dirty `gen/_mix.scss` (which is recreated on builds, so no biggie). `gen/_mix.scss` is only checked in so that theoretically, one could still build the scss without ui/build.